### ISSUE TITLE
I can't user user.save() to update the identifies of user

### ISSUE THREAD

**Issue by queyijiangnan** [python-gitlab/python-gitlab#2564]
## Description of the problem, including code/CLI snippet
I use get user and change the identifies of it,but after use user.save(),it not changes.
when i update user's name,it works. but if i update identifies, it not work.
The identifies correct ,i can update it by hand.

## Expected Behavior
after user.save() update  indetifies success

## Actual Behavior
not work,don't get any error

## Specifications

  - python-gitlab version:3.14.0
  - API version you are using (v3/v4):v4
  - Gitlab server version (or gitlab.com):15.0.1

**Comment by queyijiangnan** [comment]
`gl = gitlab.Gitlab('https://xxxx.com/', private_token='xxxx')
users = gl.users.list(all=True)
u =user[0]
u.identities[0]['provider'] ='cas3'
u.identities[0]['extern_uid'] = 'xxx'
u.save()
`

**Comment by queyijiangnan** [comment]
`gl = gitlab.Gitlab('https://xxxx.com/', private_token='xxxx')`
`users = gl.users.list(all=True)`
`u =user[0]`
`u.identities[0]['provider'] ='cas3'`
`u.identities[0]['extern_uid'] = 'xxx'`
`u.save()`

**Comment by JohnVillalovos** [comment]
I don't think that can be updated via the `users` endpoint. I don't see anyway to update it in: https://docs.gitlab.com/ee/api/users.html

Not sure but might need to do it via this endpoint: https://docs.gitlab.com/ee/api/oauth2.html
